tools/monorepo-utils/src/core/git.ts

```diff
@@ -115,6 +115,24 @@ export const cloneRepoShallow = async ( repoPath: string ) => {
 	return await cloneRepo( repoPath, { '--depth': 1 } );
 };
 
+/**
+ * Add a remote using the authenticated token `GITHUB_TOKEN`
+ *
+ * @param {Object} options       CLI options
+ * @param {string} options.owner repo owner
+ * @param {string} options.name  repo name
+ * @return {string} remote
+ */
+export const getAuthenticatedRemote = ( options: {
+	owner: string;
+	name: string;
+} ) => {
+	const { owner, name } = options;
+	const source = `github.com/${ owner }/${ name }`;
+	const token = getEnvVar( 'GITHUB_TOKEN' );
```

Comment:
Just a question, do we always want to pass `true` to this I think? to ensure it errors if it is not present?

Reply:
Yes, I think so. Fixed in https://github.com/woocommerce/woocommerce/pull/38510/commits/2072f74190d1a7bf74a3e83b810ec1245bf3c5e7